### PR TITLE
Feature: CNF Installation (5.1) Test preparation for new installation

### PR DESCRIPTION
**Pull request by barmull**
## Description
- open_metrics and prometheus_traffic - changed config map naming
- ip_addresses - removed, fully replaced by hardcoded_ip_addresses_in_k8s_runtime_configuration
- hardcoded_ip_addresses_in_k8s_runtime_configuration - change to use common_manifest.yml file

## Issues:
Refs: #2153

### CODE REVIEW

File: tools/curl_install_tester_docker_setup/Dockerfile [hunk 1]
```diff
@@ -21,4 +21,4 @@ from base as ci_test
 COPY cnf-testsuite .
 RUN ./cnf-testsuite setup verbose
 RUN wget https://raw.githubusercontent.com/cnti-testcatalog/testsuite/main/example-cnfs/coredns/cnf-testsuite.yml
-RUN ./cnf-testsuite ip_addresses cnf-config=./cnf-testsuite.yml verbose
+RUN ./cnf-testsuite ip_addresses cnf-config=./cnf-testsuite.yml verbose ?
```
Comment (martin-mat) [inline tools/curl_install_tester_docker_setup/Dockerfile:24]: I don't understand this changed line.
ip_address is being removed.
So this needs to be replaced by some different logic.

**Review by martin-mat — CHANGES_REQUESTED** [review]
Also add reasoning for removing ip_address to the commit message.

File: tools/curl_install_tester_docker_setup/Dockerfile (hunk 1 above)
Comment (martin-mat) [reply tools/curl_install_tester_docker_setup/Dockerfile:24]: replace with helm_chart_valid

**Review by kosstennbl — APPROVED** [review]
LGTM, should be ready to merge after (4), CI checks and approvals

**Review by svteb — APPROVED** [review]
lgtm

**Comment by martin-mat** [comment]
needs rebase

**Comment by barmull** [comment]
rebased on main

**Review by martin-mat — APPROVED** [review]
lgtm

**Review by collivier — APPROVED** [review]
lgtm

**Pull request merged**